src/acquisition/covidcast/csv_importer.py

```diff
@@ -1,20 +1,26 @@
 """Collects and reads covidcast data from a set of local CSV files."""
 
 # standard library
-from dataclasses import dataclass
-from datetime import date
-import glob
 import os
 import re
+from dataclasses import dataclass
+from datetime import date
+from glob import glob
+from typing import Iterator, NamedTuple, Optional, Tuple
```

Comment:
So I'm using a Python import sort extension in my IDE and it likes to place bare `import X` statements above `from X import Y` statements, but then sorts each of those, so that's convention I followed here.

Reply:
I feel like this is an open ended question that we can address in a more comprehensive push for broader linting/formatting goals. Gonna keep as is for now.